### DISCUSSION TITLE
0.3.1

### DISCUSSION THREAD

**Discussion by jmbhughes** [punch-mission/solpolpy#150] [Announcements]
## What's Changed
* fixes mzp to npol conversion by @jmbhughes in https://github.com/punch-mission/solpolpy/pull/149

**Full Changelog**: https://github.com/punch-mission/solpolpy/compare/0.3.0...0.3.1

<hr /><em>This discussion was created from the release <a href='https://github.com/punch-mission/solpolpy/releases/tag/0.3.1'>0.3.1</a>.</em>

**Comment by ddasilva** [comment]
Congrats on the release!
